cmd/fleet/handleCheckin.go

```diff
@@ -429,7 +566,7 @@ func parsePolicy(ctx context.Context, bulker bulk.Bulk, agentId string, p model.
 
 	return &resp, nil
 }
-
+*/
```

Comment:
Should this just be removed? Why leave it in and commented out?

Reply:
Oversight.  Removed it; thanks.